pkg/compute/compute_integration_test.go

```diff
@@ -346,8 +360,8 @@ func TestInit(t *testing.T) {
 	}
 }
 
-func TestBuild(t *testing.T) {
-	if os.Getenv("TEST_COMPUTE_BUILD") == "" {
+func TestBuildRust(t *testing.T) {
+	if os.Getenv("TEST_COMPUTE_BUILD_RUST") == "" && os.Getenv("TEST_COMPUTE_BUILD") == "" {
```

Comment:
Was `TEST_COMPUTE_BUILD_RUST` something that already existed previously for testing?

Reply:
No this has been added so you can run the Rust tests in isolation. See other comment RE `TEST_COMPUTE_BUILD`.